MapboxNavigation/Style.swift

```diff
@@ -196,6 +196,11 @@ public class Style: NSObject {
     public var trafficSevereColor: UIColor?
     
     /**
+     Sets an alternate color used throughout the UI to denote low traffic congestion. Not used to style the route line.
+     */
+    public var trafficAlternateLowColor: UIColor?
```

Comment:
Given that this color is used to style text, not the route line, let’s call this property `lowTrafficTextColor`.

(It would be more grammatical to call the other properties `lowTrafficColor`, `moderateTrafficColor`, etc., but I guess that can be a separate PR.)